marshaler.go

```diff
@@ -107,25 +109,38 @@ func NewEncoder(w io.Writer) *Encoder {
 //   `multiline:"true"`: when the field contains a string, it will be emitted as
 //   a quoted multi-line TOML string.
 func (enc *Encoder) Encode(v interface{}) error {
-	var b []byte
-	var ctx encoderCtx
+	var (
+		b   []byte
+		ctx encoderCtx
+	)
+
 	b, err := enc.encode(b, ctx, reflect.ValueOf(v))
 	if err != nil {
-		return err
```

Comment:
Interesting, I thought the linter would flag errors starting with an uppercase (https://github.com/golang/go/wiki/CodeReviewComments#error-strings)

Reply:
you're right, it should, and I have seen that warning many times but not for a while. Let me check on that!

Reply:
I checked, so the capital letter is not a problem if it's the function name (here Encode) or even an acronym (like TOML, for instance)

Reply:
thanks!